VirtualTools/Assets/Scripts/Instrument/Instrument.cs

```diff
@@ -26,43 +26,26 @@ public enum INSTRUMENT_TAG
     // Start is called before the first frame update
     void Start()
     {
-        m_renderer = GetComponent<MeshRenderer>();
+        m_renderer = GetComponentInChildren<MeshRenderer>();
```

Comment:
The mesh is now a child in some of the objects, so that the parent object doesn't have a non-zero rotation, since this was causing issues when instantiating the object.